### ISSUE TITLE
cc-menu-button component

### ISSUE THREAD

**Issue by florian-sanders-cc** [CleverCloud/clever-components#1106]
## Context

We recently introduced a new cc-domain-management component (see #1090) and we used two separate buttons for the primary and delete features.

We'd like to regroup them within a single button that would show a menu when triggered.

This is very close to what our `cc-popover` can already do but it should be a lot more restrictive and opiniated:

- semantics, a menu can only contain "menu items" and not checkboxes etc.,
- keyboard behaviour, arrow keys to navigation menu items, etc.,
- visual identity, the component should be different that regular buttons and different from popovers so people identify it and expect the behaviour to be different. 

The whole pattern is described in the [ARIA Design Patterns - Menu Button](https://www.w3.org/WAI/ARIA/apg/patterns/menu-button/).
